core/src/mindustry/world/blocks/production/Separator.java

```diff
@@ -30,6 +30,7 @@ public Separator(String name){
         solid = true;
         hasItems = true;
         hasLiquids = true;
+        sync = true;
```

Comment:
just push the implement suggestion button if you want to get rid of the syncing boolean:
```suggestion
```

Reply:
All GenericCrafters are synced, so it's probably fine with `sync = true`.